staging/podsecuritypolicy/rbac/README.md

```diff
@@ -32,11 +32,13 @@ It is important to note that this example uses the following syntax to test with
 
 1.  `--server=https://127.0.0.1:6443`: when performing requests this ensures that the protected port is used so
 that RBAC will be enforced
-1.  `--token={user}/{group(s)}`: this syntax allows a request to specify the username and groups to use for
-testing.  It relies on the `ALLOW_ANY_TOKEN` setting.
+1.  `--token=<token>`: this allows to make requests from a different users during testing.
 
 ## Creating the policies, roles, and bindings
 
+NOTE: If you are using `local-up-cluster.sh` you don't need to create these
```

Comment:
I don't see these created by default.

```
$ kubectl get podsecuritypolicies
No resources found.
$ kubectl get clusterrolebindings | grep psp
$
```

Reply:
They're  being created only when you provide `PSP_ADMISSION=true` option.

Here, I omit all the options because they were specified above. Let me know if you think that I should mention them explicitly.